build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
   }
   dependencies {
     classpath "org.jetbrains.dokka:dokka-gradle-plugin:$DOKKA_VERSION"
-    classpath "io.arrow-kt:ank-gradle-plugin:$ANK_GRADLE_PLUGIN_VERSION"
+    classpath "io.arrow-kt:ank-gradle-plugin:$ARROW_VERSION"
```

Comment:
Please, look at the artifact id in the repository because it changed with next versions.